draft-irtf-cfrg-rsa-blind-signatures.md

```diff
@@ -270,14 +271,17 @@ Outputs:
 Errors:
 - "unexpected input size": Raised when a byte string input doesn't
   have the expected length.
+- "invalid message length": Raised when the message representative
+  to sign is not an integer between 0 and n - 1.
 
 Steps:
 1. If len(blinded_msg) != kLen, raise "unexpected input size"
    and stop
 2. m = OS2IP(blinded_msg)
-3. s = RSASP1(skS, m)
-4. blind_sig = I2OSP(s, kLen)
-5. output blind_sig
+3. If m < n, raise "invalid message length" and stop
```

Comment:
Did you mean m >= n ?

Reply:
Hah, oops, yes. I didn't return an error before but changed it without the check.

Reply:
```suggestion
3. If m >= n, raise "invalid message length" and stop
```